src/utils/headers.test.js

```diff
@@ -70,77 +70,98 @@ test.after(async (t) => {
 /**
  * Pass if we can load the test headers without throwing an error.
  */
-test('_headers: syntax validates as expected', (t) => {
-  parseHeadersFile(path.resolve(t.context.builder.directory, '_headers'))
+test('_headers: syntax validates as expected', async (t) => {
+  await parseHeaders({ headersFiles: [path.resolve(t.context.builder.directory, '_headers')] })
 })
 
-test('_headers: throws on invalid syntax', (t) => {
-  t.throws(() => parseHeadersFile(path.resolve(t.context.builder.directory, '_invalid_headers')), {
-    message: /invalid header at line: 5/,
+test('_headers: throws on invalid syntax', async (t) => {
+  await t.throwsAsync(parseHeaders({ headersFiles: [path.resolve(t.context.builder.directory, '_invalid_headers')] }), {
+    message: /Missing header value/,
   })
 })
 
-test('_headers: validate rules', (t) => {
-  const rules = parseHeadersFile(path.resolve(t.context.builder.directory, '_headers'))
-  t.deepEqual(rules, {
-    '/': {
-      'X-Frame-Options': ['SAMEORIGIN'],
+test('_headers: validate rules', async (t) => {
+  const rules = await parseHeaders({ headersFiles: [path.resolve(t.context.builder.directory, '_headers')] })
```

Comment:
`parseHeaders()` now returns an array instead of an object.
The source code and tests were updated to take it into account.